apps/examples/testcase/le_tc/kernel/itc_pthread.c

```diff
@@ -114,7 +120,7 @@ static void *thread_set_get_specific(void *param)
 			return NULL;
 		}
 	}
-	sleep(WAIT_TIME_1);
+	pthread_key_delete(exec_index);
```

Comment:
1. exec_index -> g_key
2. Don't you need to check return value of delete?

Reply:
added return check for pthread_key_delete  API